src/App.js

```diff
@@ -22,12 +34,15 @@ export class App extends Component {
                   id="search-query"
                   className="input"
                   placeholder="Type search word"
+                  onChange={(event) => {
+                    this.setState({ query: event.target.value });
```

Comment:
```suggestion
                    this.setState({ query: event.target.value.toLowerCase() });
```
Так как это контролируемый ввод, мы можем прямо здесь приводить к нижнему регистру

Reply:
Good point